### DISCUSSION TITLE
pod airflow-run-airflow-migrations Terminated - OOMKilled

### DISCUSSION THREAD

**Discussion by lockkie0** [apache/airflow#17550] [Q&A] [answered]
Hello.
How to set resources for the migration pod? I use Helm chart v1.1.0. When I run deployment, system terminated the container as it tried to use more cpu than its limit. In values.schema.json there is no resource config for the migration pod.
```
Last State:     Terminated
Reason:       OOMKilled
Exit Code:    137
```

![image](https://user-images.githubusercontent.com/44225323/129028676-68daa59c-a540-4589-9f7e-7f68a015a80d.png)

**Comment by potiuk** [comment, marked as answer]
I believe it was OOM -> Out Of Memory not CPU. I don't think Kubernetes  ever kills a process that exceeds CPU (but It will one exceeding memory)

**Reply by lockkie0** [reply]
Yes, memory limit has been exceeded. We increased default limit and deployment completed successfully.
